File: SimpleMessengerRouter/SimpleMessengerRouter/Common/Enum.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SimpleMessengerRouter.Common
{
    class Enum
    {
        public enum ClientType
        {
            QQ, 
            Bukkit, 
            Sponge, 
            Client, 
            Bungee, 
            Console, 
            Other
        }

        public enum MsgType
        {
            msg, 
            img, 
            handshake, 
            control, 
            error
        }
    }
}

File: MinecraftForgeModServerSideOnly/src/main/java/moe/qian/forgemod/simplemessengerserversideonly/Enum.java
package moe.qian.forgemod.simplemessengerserversideonly;

public class Enum {
    public static enum Type
    {
        msg,
        img,
        handshake,
        control,
        error
    }

    public static enum From
    {
        QQ,
        Bukkit,
        Sponge,
        Client,
        Bungee,
        Console,
        Other
    }
}
